kinto_http/session.py

```diff
@@ -73,7 +73,15 @@ def request(self, method, endpoint, data=None, permissions=None,
             kwargs['headers'] = dict()
 
         if kwargs.get('params') is not None:
-            kwargs['params'] = {k: json.dumps(v).strip('"') for k, v in kwargs["params"].items()}
+            params = dict()
+            for key, value in kwargs["params"].items():
+                if key.startswith('in_') or key.startswith('exclude_'):
+                    params[key] = ','.join(value)
+                elif isinstance(value, str):
+                    params[key] = value
+                else:
+                    params[key] = json.dumps(value)
+            kwargs['params'] = params
```

Comment:
To be honest, I disagree with the version before the change and I disagree even more with the version after the change. What happens if you want to request the following records using their `message` field?

```
{
  "id": "1",
  "message": "Hello"
}
{
  "id": "2",
  "message": "\"Hello,\" says the bartender"
}
{
  "id": "3",
  "message": "\"Hello\""
}
```

Reply:
Add them to the test but I guess it should work.

Reply:
id 1:

`message='"Hello"'`

id 2:

`message='"\"Hello,\" says the bartender"'`

id 3:
`message='"\"Hello,\""'`

Reply:
I'm guessing your answer is what to pass as kwargs, but my question is about what requests are produced.